src/operation/get-parameters-as-json-schema.ts

```diff
@@ -196,25 +183,8 @@ export default function getParametersAsJsonSchema(
     return components;
   }
 
-  function getParameters(): Array<SchemaWrapper> {
-    let operationParams = operation.parameters || [];
-    const commonParams = getCommonParams();
-
-    if (commonParams.length !== 0) {
-      const commonParamsNotInParams = commonParams.filter((param: RMOAS.ParameterObject) => {
```

Comment:
This code now lives within `dedupeCommonParameters` and is run as part of `Operation.getParameters()`.